app/javascript/authentication/token.ts

```diff
@@ -63,6 +63,13 @@ export const clearHeadersTimeOut = () => {
   getStorage().removeItem(ACCESS_TOKEN_LOCAL_STORAGE_KEY)
 }
 
+export const clearHeadersConnectionIssue = () => {
+  if (getStorage().getItem(ACCESS_TOKEN_LOCAL_STORAGE_KEY)) {
+    getStorage().setItem("alert_message_secondary", t("signOut.alertMessage.connectionIssue"))
+    getStorage().removeItem(ACCESS_TOKEN_LOCAL_STORAGE_KEY)
+  }
```

Comment:
question: Why do the other `clear...` functions always remove the token (outside of the `if` statement), while this one does not?

Reply:
No reason - should function effectively the same. I can change this one for consistency though!